plugins/avro-format/core/src/main/java/org/pentaho/di/trans/steps/avro/AvroToPdiConverter.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Big Data
  *
- * Copyright (C) 2022 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2023 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
Copyright date should read 2022 - 2023 here.  Looks ok otherwise.

Reply:
Got it I will make the changes and squash